### PR TITLE
[WIP] additional configure flags

### PR DESCRIPTION
**Pull request by rwhogg**
See #2 .

All of the formulae tried so far accept `--disable-silent-rules` and `--disable-dependency-tracking`, but not `--disable-debug`.
## TODO
- [ ] Rebuild all the formulae from scratch once I've updated them all.

### CODE REVIEW

**Comment by maxim-belkin** [comment]
Cool! Thanks! Does `xorg` build fine with these rules?

**Comment by bw-yuhogg** [comment]
Haven't been able to check that yet. I'll make sure to do so before removing the "WIP" from the title. (Adding a TODO to the summary to remind myself)

It is _really_ tedious to do this because of how many libraries there are, so it may take a while.

File: xcb-proto.rb [hunk 1]
```diff
@@ -25,6 +25,7 @@ def install
       --prefix=#{prefix}
       --sysconfdir=#{etc}
       --localstatedir=#{var}
+      --disable-silent-rules
```
Comment (bw-yuhogg) [inline xcb-proto.rb:28]: Yes, this formula takes `--disable-silent-rules` but not `--disable-dependency-tracking`.

**Comment by maxim-belkin** [comment]
Alright. I'm going to merge these and run a test suite on the updated formulas (I've automated it via a script).

**Comment by bw-yuhogg** [comment]
This was not quite done yet, there were still some more formulae I had yet to check. 

On March 14, 2016 12:11:31 PM PDT, Maxim Belkin notifications@github.com wrote:

> Merged #5.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/Linuxbrew/homebrew-xorg/pull/5#event-589229579

## 

Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by maxim-belkin** [comment]
Now you can push, so commit whenever you are done with any single formula.

Side note: I've removed a dependency on autoconf from the formulas as we don't actually need it. I believe it is needed only when `autoconf` or `autoreconf` are used. I'm testing the change right now.

**Pull request merged**